### PR TITLE
Milestone 5

### PR DESCRIPTION
**Pull request by bouaik**
## In this Milestone
- Implemented the handleClick function
- Attached an onClick event to all buttons.
- Passed the clickHandler property to each button in the panel.
- Modify logic

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hi @bouaik 👋

**You've done quite well on this. 👍**

- [ ] However, kindly correct this bug. Disable clicking the `equal(=)` button as the first click to avoid this issue👇:
ice_screenshot_20201006-093801

- [ ] OPTIONAL - Consider using an arrow function to setState.

Kindly fix this issue and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: src/components/ButtonPanel.js [hunk 1]
```diff
@@ -1,40 +1,45 @@
...
+        <Button name="+" clickHandler={clickHandler} />
       </div>
       <div className="group">
-        <Button name="0" color="#dfdfdf" wide />
-        <Button name="." color="#dfdfdf" />
-        <Button name="=" />
+        <Button name="0" color="#dfdfdf" clickHandler={clickHandler} wide />
+        <Button name="." color="#dfdfdf" clickHandler={clickHandler} />
+        <Button name="=" clickHandler={clickHandler} />
       </div>
```
Comment (MauricioRobayo) [inline src/components/ButtonPanel.js:36]: **OPTIONAL**: A lot of duplicated code 🤔, do you think we could do this without repeating ourselves so much?

**Review by MauricioRobayo — APPROVED** [review]
Bug fixed. This is it, you are ready for bigger challenges, please move on and keep up the excellent work.

Status: **Approved**.

File: src/components/ButtonPanel.js (hunk 1 above)
Comment (bouaik) [reply src/components/ButtonPanel.js:36]: Yes, I can create a function that returns the button component and pass name and color as arguments.

**Pull request merged**